.github/workflows/release_binaries.yml

```
@@ -0,0 +1,32 @@
+name: "Release Binaries"
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - master
+    types: [closed]
```

```suggestion
    types:
      - closed
```
YAML array consistency nit